$NetBSD: patch-aa,v 1.1.1.1 2010/09/21 11:47:32 fhajny Exp $

Makefile really expects Bash to be used for shell calls

--- Makefile.orig	2009-10-06 14:45:47.000000000 +0000
+++ Makefile
@@ -1,4 +1,4 @@
-
+SHELL=@SHELL@
 TMPDIR ?= /tmp
 
 RABBITMQ_NODENAME ?= rabbit
